<!DOCTYPE html>
<html lang="en">
<head>
<meta charset="utf-8">
<title>Quinlan Lab @UVa</title>
<meta name="viewport" content="width=device-width, initial-scale=1.0">
<meta name="description" content="">
<meta name="author" content="">
<!-- Le styles -->
<link rel="stylesheet/less" href="docs/assets/less/variables.less">
<script src="js/less.js"></script>
<link href="css/bootstrap.min.css" rel="stylesheet">
<style type="text/css">
    body {
        padding-top: 60px;
        padding-bottom: 40px;
    }
</style>
<link href="css/bootstrap-responsive.css" rel="stylesheet">
<!-- Le HTML5 shim, for IE6-8 support of HTML5 elements -->
<!--[if lt IE 9]>
    <script src="//html5shim.googlecode.com/svn/trunk/html5.js"></script>
    <![endif]-->
<!-- Le fav and touch icons -->
<link rel="shortcut icon" href="images/favicon.ico">
<link rel="apple-touch-icon" href="images/apple-touch-icon.png">
<link rel="apple-touch-icon" sizes="72x72" href="images/apple-touch-icon-72x72.png">
<link rel="apple-touch-icon" sizes="114x114" href="images/apple-touch-icon-114x114.png">

<script type="text/javascript">

  var _gaq = _gaq || [];
  _gaq.push(['_setAccount', 'UA-34093789-1']);
  _gaq.push(['_trackPageview']);

  (function() {
    var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
    ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
  })();

</script>
</head>
<body>
    
    <div class="navbar navbar-fixed-top">
        <div class="navbar-inner">
            <div class="container">
                <a class="btn btn-navbar" data-toggle="collapse" data-target=".nav-collapse">
                    <span class="icon-bar"></span>
                    <span class="icon-bar"></span>
                    <span class="icon-bar"></span>
                </a>
                <a class="brand" href="index.html">Quinlan Lab @UVA</a>
                <div class="nav-collapse">
                    <ul class="nav">
                        <li><a href="index.html">Home</a></li>
                        <li class="active"><a href="research.html">Research</a></li>
                        <li><a href="publications.html">Publications</a></li>
                        <li><a href="software.html">Software</a></li>
                        <li><a href="teaching.html">Teaching</a></li>
                        <li><a href="people.html">People</a></li>
                        <li><a href="contact.html">Contact</a></li>
                        <li><a href="http://geneticcode.tumblr.com/">Blog</a></li>
                    </ul>
                </div><!--/.nav-collapse -->
            </div>
        </div>
    </div>


<div class="container">

    <div>
        <p class="lead">
                The research in our laboratory is largely focused on developing 
                and applying computational methods 
                towards the understanding of genetic variation in diverse 
                contexts. Modern experimental methods allow 
                us to examine entire genomes with exquisite detail. Perhaps not 
                surprisingly, staggering complexity 
                is revealed as we look more closely at genome structure and the 
                landscape of genetic variation. The 
                throughput of modern genomic technologies (such as 
                next-generation DNA sequencing) necessitates 
                efficient approaches for manipulating and comparing large 
                genomic datasets and for characterizing 
                the genetic variation therein. Our laboratory develops such 
                methods so that we and others may apply 
                them to experiments investigating the impact of genetic 
                variation on human disease, evolution, and 
                somatic differentiation. We are actively involved in several 
                collaborations including: exome 
                sequencing in multiple disease phenotypes, screens for rare 
                mutations and copy-number variation 
                in Type 1 diabetes, characterization of hundreds of cancer 
                genomes, and the genomics of induced 
                pluripotent stem (iPS) cells.
        </p>
    </div>

    
    <div class="row"  style="margin-top:10px;padding-top:20px">
        <div class="span8">
            <h2><b>Computational Genomics</b></h2>
        </div>
    </div>
    <div class="row">
        <div class="span8">
            <p>
                Technological advances have made genomics an intensely 
                computational field of research. A 
                pressing challenge is the fact that we can sequence entire 
                genomes much more rapidly than we can <em>interpret</em> them. 
                Consequently, many geneticists struggle to apply modern 
                molecular techniques to their specific area of expertise. 
                Moreover, the scale and inherent complexity of genomic 
                datasets complicates even the most basic data exploration and
                interpretation. Our laboratory develops pragmatic and 
                <a href="software.html">
                    innovative computer software 
                </a>
                for manipulating and interpreting 
                massive genomics datasets. We 
                currently develop software in three main areas: 
                <strong>(1)</strong> genome 
                data mining and analysis, <strong>(2)</strong> discovery, 
                annotation, and interpretation of genetic variation, and 
                <strong>(3)</strong> rapid comparison of complete 
                genome sequences.
            </p>
        </div>
        <div class="span4">
            <ul class="thumbnails">
                <li>
                <a href="#">
                <img src="img/research/ppi.jpg" border="0">
                </a>
                </li>
            </ul>
        </div>
    </div>


    <div class="row">
        <div class="span8">
            <h2><b>Variation in genome structure</b></h2>
        </div>
    </div>
    <div class="row">
        <div class="span8">
            <p>
            Human chromosomes harbor hundreds of structural differences 
            including deletions, insertions, duplications, inversions, and 
            translocations. Collectively, these differences are known as 
            "structural variation" (or, "SV"). Any two humans differ by 
            thousands of structural variants which vary greatly in size and 
            phenotypic consequence. Thanks to experimental 
            advances and large-scale studies such as the 
            <a href="http://www.1000genomes.org">1000 Genomes Project</a>, 
            our understanding of the prevalence of SV and the contribution of 
            each mechanism to SV in the germline has increased dramatically. 
            However, we are just beginning to understand the contribution of SV 
            to evolution, development, and complex disease.Our laboratory 
            continues to develop new 
            methods for detecting and understanding structural variation using 
            modern DNA sequencing techniques.
            </p>
        </div>
        <div class="span4">
            <ul class="thumbnails">
                <li>
                <a href="#">
                <img src="img/research/sv.png" border="0">
                </a>
                </li>
            </ul>
        </div>
    </div>


    <h2><b>Somatic genome evolution and cancer</b></h2>

    <div class="row">
        <div class="span8">
            <p>
            Over the lifetime of an individual, the genomes of somatic cell 
            lineages acquire many structural changes that rearrange their 
            genetic material. Unlike point mutations such as single-nucleotide 
            polymorphisms (SNP), structural lesions have the potential to 
            amplify, remove, or reconstruct entire genes. The number and impact 
            of these mutations depends on the age and genetic background of the 
            individual, the cell type in question, and exposure to mutagens. Our 
            laboratory is interested in several questions related to 
            somatically-acquired mutation. First, we want to understand whether 
            specific regions of our genome are "hotspots" for rearrangement; 
            that is, are they more prone to breakage and rearrangement then 
            others? Second, are structural rearrangements so-called "driver" 
            mutations in cancer? Lastly, the structural integrity of founding 
            somatic lineages is likely to have direct consequences on the 
            development of stable, patient-specific, induced pluripotent stem 
            (iPS) cell lineages for therapeutic applications.
            </p>
        </div>
        <div class="span4">
            <ul class="thumbnails">
                <li>
                <a href="#">
                <img src="img/research/cancer.png" border="0">
                </a>
                </li>
            </ul>
        </div>
    </div>
    <hr>
    <footer>
    <p>
        &copy; Quinlan Lab @UVA
    </p>
    </footer>
</div>
<!-- /container -->
<!-- Le javascript
    ================================================== -->
<!-- Placed at the end of the document so the pages load faster -->
<script src="js/jquery.js"></script>
<script src="js/bootstrap-transition.js"></script>
<script src="js/bootstrap-alert.js"></script>
<script src="js/bootstrap-modal.js"></script>
<script src="js/bootstrap-dropdown.js"></script>
<script src="js/bootstrap-scrollspy.js"></script>
<script src="js/bootstrap-tab.js"></script>
<script src="js/bootstrap-tooltip.js"></script>
<script src="js/bootstrap-popover.js"></script>
<script src="js/bootstrap-button.js"></script>
<script src="js/bootstrap-collapse.js"></script>
<script src="js/bootstrap-carousel.js"></script>
<script src="js/bootstrap-typeahead.js"></script>
</body>
</html>