<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="utf-8">
	<title>Quinlan Lab @UVa</title>
	<meta name="viewport" content="width=device-width, initial-scale=1.0">
	<meta name="description" content="">
	<meta name="author" content="">

	<!-- Le styles -->
	<link href="css/bootstrap.min.css" rel="stylesheet">
	<style type="text/css">
	body {
		padding-top: 60px;
		padding-bottom: 40px;
	}
	</style>
	<link href="css/bootstrap-responsive.css" rel="stylesheet">

	<!-- Le HTML5 shim, for IE6-8 support of HTML5 elements -->
	<!--[if lt IE 9]>
	<script src="//html5shim.googlecode.com/svn/trunk/html5.js"></script>
	<![endif]-->

	<!-- Le fav and touch icons -->
	<link rel="shortcut icon" href="images/favicon.ico">
	<link rel="apple-touch-icon" href="images/apple-touch-icon.png">
	<link rel="apple-touch-icon" sizes="72x72" href="images/apple-touch-icon-72x72.png">
	<link rel="apple-touch-icon" sizes="114x114" href="images/apple-touch-icon-114x114.png">
	
	<script type="text/javascript">

  var _gaq = _gaq || [];
  _gaq.push(['_setAccount', 'UA-34093789-1']);
  _gaq.push(['_trackPageview']);

  (function() {
    var ga = document.createElement('script'); ga.type = 'text/javascript'; ga.async = true;
    ga.src = ('https:' == document.location.protocol ? 'https://ssl' : 'http://www') + '.google-analytics.com/ga.js';
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(ga, s);
  })();

  </script>
</head>

<body>

  <div class="navbar navbar-fixed-top">
    <div class="navbar-inner">
      <div class="container">
        <a class="btn btn-navbar" data-toggle="collapse" data-target=".nav-collapse">
          <span class="icon-bar"></span>
          <span class="icon-bar"></span>
          <span class="icon-bar"></span>
        </a>
        <a class="brand" href="index.html">Quinlan Lab @UVA</a>
        <div class="nav-collapse">
          <ul class="nav">
            <li><a href="index.html">Home</a></li>
            <li><a href="research.html">Research</a></li>

            <li class="dropdown">
              <a href="publications.html" class="dropdown-toggle" data-toggle="dropdown">Publications <b class="caret"></b></a>
              <ul class="dropdown-menu">
                <li><a href="publications.html#Manuscripts">Manuscripts</a></li>
                <li><a href="publications.html#Lectures">Lectures</a></li>
              </ul>
            </li>


            <li class="active"><a href="software.html">Software</a></li>
            <li><a href="teaching.html">Teaching</a></li>
            <li><a href="people.html">People</a></li>
            <li><a href="contact.html">Contact</a></li>
            <li><a href="http://geneticcode.tumblr.com/">Blog</a></li>
          </ul>
        </div><!--/.nav-collapse -->
      </div>
    </div>
  </div>

  <div class="container-fluid">
    <div class="row-fluid">
      <div class="span2">
        <div class="well sidebar-nav">
          <ul class="nav nav-list">

            <li class="nav-header" style="font-size: 12pt;">bedtools</li>
            <li style="font-size: 10pt;"><a 
              href="https://code.google.com/p/bedtools/">Details</a></li>
              <li style="font-size: 10pt;"><a
                href="https://bedtools.googlecode.com/files/BEDTools-User-Manual.v4.pdf">Docs</a
                ></li>
                <li style="font-size: 10pt;">
                  <a 
                  href="https://groups.google.com/group/bedtools-discuss/">Support</a></li>
                  
                  <li class="nav-header" style="font-size: 12pt;">pybedtools</li>
                  <li style="font-size: 10pt;"><a 
                    href="https://github.com/daler/pybedtools">Details</a></li>
                    <li style="font-size: 10pt;"><a
                      href="http://pythonhosted.org/pybedtools/">Docs</a
                      ></li>
                      <li style="font-size: 10pt;">
                        <a 
                        href="https://groups.google.com/group/bedtools-discuss/">Support</a></li>

                  <li class="nav-header" style="font-size: 12pt;">gemini</li>
                  <li style="font-size: 10pt;">
                    <a href="https://github.com/arq5x/gemini">Details</a>
                  </li>
                  <li style="font-size: 10pt;">
                    <a href="https://github.com/arq5x/gemini">Docs</a>
                  </li>

                  <li class="nav-header" style="font-size: 12pt;">hydra</li>
                  <li style="font-size: 10pt;"><a href="http://code.google.com/p/hydra-sv/">Details</a></li>
                  <li style="font-size: 10pt;"><a href="http://code.google.com/p/hydra-sv/#Documentation">Docs</a></li>

                  <li class="nav-header" style="font-size: 12pt;">lumpy</li>
                  <li style="font-size: 10pt;">
                    <a href="https://github.com/arq5x/lumpy-sv">Details</a>
                  </li>

                  <li class="nav-header" style="font-size: 12pt;">scurgen</li>
                  <li style="font-size: 10pt;">
                    <a href="https://github.com/arq5x/scurgen">Details</a>
                  </li>

                </ul>
              </div><!--/.well -->
            </div><!--/span-->
            <div class="span8">
              <p>
                <font size="4">
                  Our laboratory develops free, open-source software for genomics
                  research. We strive to develop intuitive and well-documented 
                  tools and are always open to feedback and user requests. If
                  you find these tools useful in your research, we ask that
                  you cite them in your research and report any issues that you
                  uncover.
                </font>
              </p>

              <div class="row-fluid"
              id="bedtools">
              <h3>bedtools</h3>
              <div class="span10">
                <p>
                  <em>Bedtools</em> is a flexible toolset for genome 
                  arithmetic. It
                  supports a wide range of common data manipulation tasks
                  for genomic intervals (e.g. genes, ChIP-seq peaks) in
                  many commonly used genomics file formats (
                  BED, BAM, GFF, VCF).
                </p>
              </div><!--/span-->
              <div class="btn-group span1">
                <a class="btn btn-primary dropdown-toggle " data-toggle="dropdown" href="#">
                  More
                  <span class="caret"></span>
                </a>
                <ul class="dropdown-menu">
                  <a href="https://code.google.com/p/bedtools/downloads/list">Download</a>
                </ul>
              </div>
            </div><!--/row-->
            
            <div class="row-fluid"
            id="pybedtools">
            <h3>pybedtools</h3>
            <div class="span10">
              <p>
                <em>Pybedtools</em> is a Python wrapper (and much more) for Bedtools 
                and extends these "genome algebra" programs by offering feature-level 
                manipulations from with Python.  Pybedtools is maintained by Ryan Dale.
              </p>
            </div><!--/span-->
            <div class="btn-group span1">
              <a class="btn btn-primary dropdown-toggle " data-toggle="dropdown" href="#">
                More
                <span class="caret"></span>
              </a>
              <ul class="dropdown-menu">
                <a href="http://pythonhosted.org/pybedtools/">Download</a>
              </ul>
            </div>
          </div><!--/row-->
        

            <div class="row-fluid"
            id="gemini">
            <h3>gemini</h3>
            <div class="span10">
              <p>
                <em>Gemini</em> is a powerful framework for exploring
                genetic variation in the context of the wealth of
                existing genome annotations that are available for the
                human genome. By integrating diverse annotations with
                genetic variation in the now standard 
                <a href="http://www.1000genomes.org/wiki/Analysis/Variant%20Call%20Format/vcf-variant-call-format-version-41">VCF</a> format, researchers
                have an single system for prioritizing variants in studies
                of human disease.
              </p>
            </div><!--/span-->
            <div class="btn-group span1">
              <a class="btn btn-primary dropdown-toggle " data-toggle="dropdown" href="#">
                More
                <span class="caret"></span>
              </a>
              <ul class="dropdown-menu">
                <a href="https://github.com/arq5x/gemini">Download</a>
                <a href="pdf/Gemini-Quinlan-Stroke-UVA.pdf">Slides</a>
              </ul>
            </div>
          </div><!--/row-->
          
          <div class="row-fluid"
          id="hydra">
          <h3>hydra</h3>
          <div class="span10">
            <p>
              <em>Hydra</em> detects all classes of structural variation
              using paired-end sequence alignments from modern
              DNA sequencing technologies. Unlike other existing
              tools, <em>hydra</em> detects SVs arising from repetitive
              DNA and can make improved SV predictions by incorporating
              sequence alignments from many (100s or more) samples.
            </p>
          </div><!--/span-->
          <div class="btn-group span1">
            <a class="btn btn-primary dropdown-toggle " data-toggle="dropdown" href="#">
              More
              <span class="caret"></span>
            </a>
            <ul class="dropdown-menu">
              <a href="http://code.google.com/p/hydra-sv/">Download</a>
            </ul>
          </div>
        </div><!--/row-->

        <div class="row-fluid" id="lumpy">
          <h3>lumpy</h3>
          <div class="span10">
            <p>
              <em>Lumpy</em> is a new probabilistic framework that
              we have developed to integrate multiple structural 
              variation signals such as discordant paired-end alignments
              and split-read alignments.  While it is clear that
              integrating all SV signals is important for sensitive
              discovery, most existing (including our own Hydra) tools 
              only exploit one signal. <em>Lumpy</em> integrates 
              multiple signals in order to improve sensitivity and 
              breakpoint resolution. This is especially important for
              cancer genome analysis where tumor heterogeneity causes
              potentially important rearrangements occur with less
              supporting alignments in the sampled DNA.
            </p>
          </div><!--/span-->
          <div class="btn-group span1">
            <a class="btn btn-primary dropdown-toggle " data-toggle="dropdown" href="#">
              More
              <span class="caret"></span>
            </a>
            <ul class="dropdown-menu">
              <a href="https://github.com/arq5x/lumpy-sv">Download</a>
              <a href="presentations/lumpy/lumpy-reveal-js/index.html">Slides</a>
            </ul>
          </div>
        </div><!--/row-->

        <div class="row-fluid" 
        id="scurgen">
        <h3>scurgen</h3>
        <div class="span10">
          <p>
            <em>Scurgen</em> is a new visualization tool we are
            developing for exploring genomic data with space-filling
            curves.  Nothing is better at pattern detection than 
            the human eye. <em>Scurgen</em> attempts to present
            genomic data is a way such that patterns are easily
            detected among multiple datasets and it works nicely with
            BAM, BED, VCF, BEDGRAPH, and GFF formats.
          </p>
        </div><!--/span-->
        <div class="btn-group span1">
          <a class="btn btn-primary dropdown-toggle " data-toggle="dropdown" href="#">
            More
            <span class="caret"></span>
          </a>
          <ul class="dropdown-menu">
            <a href="https://github.com/arq5x/scurgen">Download</a>
          </ul>
        </div>
      </div><!--/row-->


      <!-- /container -->
	  <!-- Le javascript
   ================================================== -->
   <!-- Placed at the end of the document so the pages load faster -->
   <script src="js/jquery.js"></script>
   <script src="js/bootstrap-transition.js"></script>
   <script src="js/bootstrap-alert.js"></script>
   <script src="js/bootstrap-modal.js"></script>
   <script src="js/bootstrap-dropdown.js"></script>
   <script src="js/bootstrap-scrollspy.js"></script>
   <script src="js/bootstrap-tab.js"></script>
   <script src="js/bootstrap-tooltip.js"></script>
   <script src="js/bootstrap-popover.js"></script>
   <script src="js/bootstrap-button.js"></script>
   <script src="js/bootstrap-collapse.js"></script>
   <script src="js/bootstrap-carousel.js"></script>
   <script src="js/bootstrap-typeahead.js"></script>
 </body>
 </html>